exasol_advanced_analytics_framework/interface/create_event_handler_udf.py

```diff
@@ -1,44 +1,93 @@
-from exasol_advanced_analytics_framework.event_handler.event_handler_base import \
-    EventHandlerBase
-from exasol_advanced_analytics_framework.event_handler.event_handler_context import EventHandlerContext
 import importlib
-
-from exasol_advanced_analytics_framework.event_handler.event_handler_result import \
-    EventHandlerResult
+from collections import OrderedDict
+from pathlib import PurePosixPath
+from exasol_bucketfs_utils_python.bucketfs_factory import BucketFSFactory
+from exasol_bucketfs_utils_python.bucketfs_location import BucketFSLocation
+from exasol_advanced_analytics_framework.event_handler.event_handler_base \
+    import EventHandlerBase
+from exasol_advanced_analytics_framework.event_handler.event_handler_context \
+    import EventHandlerContext
+from exasol_advanced_analytics_framework.event_handler.event_handler_result \
+    import EventHandlerResult
+from exasol_advanced_analytics_framework.event_handler.event_handler_state \
+    import EventHandlerState
+from exasol_advanced_analytics_framework.utils.udf_context_wrapper import \
+    UDFContextWrapper
 
 
 class CreateEventHandlerUDF:
+    STATE_DIRECTORY = "aaf_state"
+
     def __init__(self, exa):
         self.exa = exa
 
     def run(self, ctx) -> None:
+        # get and set method parameters
         parameters = ctx.parameters
         bucketfs_connection = ctx.bucketfs_connection
+        bucketfs_location = BucketFSFactory().create_bucketfs_location(
+            url=bucketfs_connection.address,
+            user=bucketfs_connection.user,
+            pwd=bucketfs_connection.password,
+            base_path=parameters["base"]
+        )
+        bucketfs_path = PurePosixPath(
+            self.STATE_DIRECTORY, f"{parameters['bucket_file_path']}.pkl")
+
+        # load the latest (create if not) event handler state object
+        latest_state = self._load_latest_state(
+            parameters, bucketfs_location, bucketfs_path)
+        event_handler_context: EventHandlerContext = latest_state.context
+        event_handler: EventHandlerBase = latest_state.event_handler
 
-        event__handler_context = EventHandlerContext(bucketfs_connection)
-        event_handler_obj = self._get_event_handler_obj(
-            parameters,  event__handler_context)
-        result: EventHandlerResult = event_handler_obj.handle_event()  # TODO ?
-        self._save_event_handler_obj(event_handler_obj, event__handler_context)
+        # call the user code
+        udf_context = self._create_udf_context_wrapper(ctx)
+        result: EventHandlerResult = event_handler.handle_event(
+            udf_context, event_handler_context)  # TODO does user change state?
```

Comment:
the attributes of the `event_handler` and `event_handler_context` will be changed after calling `handle_event`